app/components/common/Feedback.tsx

```diff
@@ -257,7 +223,7 @@ What’s happening now?
 const FORM_ERRORS: Partial<FormFields> = {
   name: 'Your name should not be empty',
```

Comment:
if you decide to go on with only discord handle, then this also should be updated